privacyidea/lib/token.py

```diff
@@ -1824,6 +1826,8 @@ def lost_token(serial, new_serial=None, password=None,
             character_pool += string.digits
         if "s" in contents:
             character_pool += "!#$%&()*+,-./:;<=>?@[]^_"
+        if "8" in contents:
+            character_pool += BASE58
```

Comment:
Might be a bit far-fetched, but: If someone enters ``8C`` or so for the character pool, this will result in some chars, e.g. ``A``, being duplicated in ``character_pool``, which will result in those chars being more likely to be part of the password than others. Maybe it would be a good idea to add a note to the documentation saying that ``8`` should not be combined with other chars?

Reply:
You are totally right! 
Isn't his an interesting feature being able to do 8C? But you could do 8S, which would make totally sense.
I think this is totally fine to have shit-in-shit-out ;-)

Reply:
OK, I add a docu!

Reply:
We could "uniquify" the `character_pool` but this still leaves us with the `l` and `I` issue.

Reply:
No, we do not do this. There is no sense in overengineering at this point. The documentation was added in 6bb933d92a39d55f1890a8c5bfd63181ac55fff9 and the system will behave as configured.